.github/workflows/pr-check.yml

```
@@ -0,0 +1,33 @@
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+env:
+  WORKING_DIR: apps
+
+jobs:
+  analyze-and-format:
+    name: Analyze and Format Check
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)